configs/config.toml

```
@@ -7,21 +7,35 @@ tls_certificate = "/app/ssl/tls.pem"
 tls_key = "/app/ssl/tls.key"
 # tls_client_auth = "/app/ssl/trust_anchor.pem"
 
-[pet]
-min_sum_count = 1
-min_update_count = 3
-min_sum2_count = 1
-max_sum_count = 100
-max_update_count = 10000
-max_sum2_count = 100
-min_sum_time = 5
-min_update_time = 10
-min_sum2_time = 5
-max_sum_time = 3600
-max_update_time = 3600
-max_sum2_time = 3600
-sum = 0.5
-update = 0.9
+[pet.sum]
+prob = 0.5
+
+[pet.sum.count]
+min = 1
+max = 100
+
+[pet.sum.time]
+min = 5
+max = 3600
+
+[pet.update]
+prob = 0.9
+
+[pet.update.count]
+min = 3
+max = 10000
+
+[pet.update.time]
+min = 10
+max = 3600
+
+[pet.sum2.count]
+min = 1
+max = 100
+
+[pet.sum2.time]
+min = 5
+max = 3600
```

I would prefer it if it is not so verbose

```suggestion
[pet.sum]
prob = 0.5
count = { min = 1, max = 100 }
time = { min = 5, max = 3600 }

[pet.update]
prob = 0.9
count = { min = 3, max = 10000 }
time = { min = 10, max = 3600 }

[pet.sum2]
count = { min = 1, max = 100 }
time = { min = 5, max = 3600 }
```

---

i didn't know that this syntax is possible, that definitely ways more concise 👍 i'll change that.